py3xui/async_api/async_api.py

```diff
@@ -75,14 +93,22 @@ def session(self, value: str) -> None:
         self.database.session = value
 
     @classmethod
-    def from_env(cls, logger: Any | None = None) -> AsyncApi:
+    def from_env(
```

Comment:
Should we store those parameters in envvars?
It would be nice if we still have an option to create an instance for API object without any additional arguments.
By, the way, since those two are optional (in some cases) we can also have both options: pass as arguments OR read from env. So if they're not passed let's try to find them in envvars.
Same for sync Api.